tests/unit/CMakeLists.txt

```diff
@@ -426,3 +426,14 @@ target_link_libraries(test_PipelineCTracing revngPipelineC revngSupport
                       Boost::unit_test_framework ${LLVM_LIBRARIES})
 revng_add_test(NAME test_PipelineCTracing COMMAND test_PipelineCTracing)
 set_tests_properties(test_PipelineCTracing PROPERTIES LABELS "unit")
+
+#
```

Comment:
Can we have a bash test that checks files can be uncompressed with `tar`?

Reply:
Added